rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -251,7 +252,6 @@ SettingsDialog::SettingsDialog(wxWindow *parent, rpcs3::config_t* cfg)
 		chbox_gs_overlay->Enable(false);
 	}
 
-	cbox_gs_render->Append("Vulkan");
```

Comment:
Its not as simple as compiling and running on linux: one has to set up the X11 setup for vulkan. Its not much work, but it will not run out of the box just yet.